charts/prefect-server/values.schema.json

```diff
@@ -42,7 +42,7 @@
           "type": "object",
           "title": "Prefect",
           "description": "global prefect configuration",
-          "additionalProperties": false,
+          "additionalProperties": true,
```

Comment:
this allows for the validation helper to be the one to fail the helm install/upgrade, rather than the json schema. It will provide more direct guidance.